### PR TITLE
Add option to view log in explorer. Ref #4797

### PR DESCRIPTION
**Pull request by fernandreu**
This also modifies the copyright year in the bottom label so that it automatically updates (as @retailcoder suggested). Interestingly, the ViewModel was ready for that, but the string resources weren't (they had a hard-coded `2018` instead of `{0}`.

A few extra notes:

1. I also changed the `GitHubIssueUrl` so that now it gives you the option to choose the bug template
2. I thought the trailing dot in the copy version label looked a bit weird with the hyperlink style, so I removed it
3. The code to obtain the path to the log file is taken from [here](https://stackoverflow.com/a/11629408/1712861)
4. I just used Google Translate for the Czech and French translations (which means I am not entirely sure of what I have written there!)

### CODE REVIEW

File: Rubberduck.Core/UI/About/AboutControlViewModel.cs [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 namespace Rubberduck.UI.About
 {
+    using NLog.Targets;
```
Comment (bclothier) [inline Rubberduck.Core/UI/About/AboutControlViewModel.cs]: The convention has been to put the `using` at the top of file, outside of the `namespace`.

File: Rubberduck.Core/UI/About/AboutControl.xaml [hunk 2]
```diff
@@ -99,7 +106,7 @@
                     <Border Style="{StaticResource ReportButtonBorder}">
                         <TextBlock>
                             <Hyperlink Name="GitHubIssueUrl"
-                                       NavigateUri="https://github.com/rubberduck-vba/Rubberduck/issues/new"
+                                       NavigateUri="https://github.com/rubberduck-vba/Rubberduck/issues/new/choose"
```
Comment (comintern) [inline Rubberduck.Core/UI/About/AboutControl.xaml]: Loving this new link. We should probably add at least a "feature request" template on GH to use with.

File: Rubberduck.Core/UI/About/AboutControlViewModel.cs [hunk 3]
```diff
@@ -47,6 +49,29 @@ public CommandBase UriCommand
             }
         }
 
+        private CommandBase _viewLogCommand;
```
Comment (comintern) [inline Rubberduck.Core/UI/About/AboutControlViewModel.cs]: Commands bound to WPF views probably shouldn't be lazily instantiated - it makes the `EvaluateCanExecute` bindings behave oddly.

Comment (fernandreu) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: For my own projects, what I would typically do is have no private field, just:

    public AboutControlViewModel(IVersionCheck version)
    {
        _version = version;
        ViewLogCommand = new DelegateCommand(...);
    }
    
    ...
    
    public CommandBase ViewLogCommand { get; }

Is that more in the lines of what you were thinking? If so, should I do the same with `UriCommand`?

Comment (comintern) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: That's the route I'd go (and yeah, if you want to catch the `UriCommand` too that would be great.  Given what the dialog actually _does_, this is mainly a nitpick but I've been trying to clean these up as I run across them.

**Comment by fernandreu** [comment]
No problem! I have now removed the Czech and French translations, and have put that import line outside the namespace (this was mostly due to my ReSharper / StyleCop settings, which I still need to configure for this project).

File: Rubberduck.Core/UI/About/AboutControlViewModel.cs [hunk 4]
```diff
@@ -30,24 +34,29 @@ public AboutControlViewModel(IVersionCheck version)
 
         public string HostExecutable => string.Format(AboutUI.AboutWindow_HostExecutable,
             Path.GetFileName(Application.ExecutablePath).ToUpper()); // .ToUpper() used to convert ExceL.EXE -> EXCEL.EXE
+            
+        public string AboutCopyright =>
+            string.Format(AboutUI.AboutWindow_Copyright, DateTime.Now.Year);
```
Comment (retailcoder) [inline Rubberduck.Core/UI/About/AboutControlViewModel.cs]: Sorry for dancing around this, but @mansellan and I had a [discussion](https://chat.stackexchange.com/transcript/message/49364390#49364390) following this suggestion, and this was the killer argument:

> I suspect that DateTime.Now.Year is out, because if\when RD becomes abandonware it's unjustifiable to claim copyright until the end of time...

I didn't think this through when I suggested using `DateTime.Now.Year` to just make the update automatic: even if RD keeps being maintained until the end of time, it's not reasonable to have a given build suddenly (on Jan 1st) assert copyright for code that isn't in *that* build.

So bottom line, maybe it's better to simply keep updating the resource manually every year to assert copyright of our collective contributions for that year. That said, it doesn't have to be fixed in *this PR*, or even at any point this year -- just something to keep in mind.

Comment (bclothier) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: Just to point out - we already have a build task to update the copyright year. No reason we can't do the same for this case.

Comment (fernandreu) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: Just to double-check: are you guys happy with me leaving the copyright year as it currently is in this PR (with the idea of fixing that on the build system in a separate PR later), or should I do any modifications already?

Comment (bclothier) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: By me, it's OK to merge the PR as-is; I have to agree that the copyright year and all that is technically out of scope of the PR and would likely go beyond the UI changes this PR focuses on.

Comment (Vogel612) [reply Rubberduck.Core/UI/About/AboutControlViewModel.cs]: Considering how long this PR has been approved and how lacking the response to this very justified question has been, I've gone ahead and merged the PR. Dealing with this may as well be a separate PR
